Scripting-Engine/Scripting-Engine/LeagueSandbox-Default/Champions/Ezreal/E.cs

```diff
@@ -22,54 +23,44 @@ static void onFinishCasting(Champion owner, Spell spell)
             Vector2 trueCoords;
 
             if (to.Length() > 475) {
-                //to = to.Normalize()
-                //local range = to * 475
-                //trueCoords = current:copy() + range
-            } else {
-                trueCoords = new Vector2(spell.X, spell.Y);
-             }
-            /*
-
-    if to:length() > 475 then
-        to = to:normalize()
-                local range = to * 475
-                trueCoords = current:copy() + range
+                to = Vector2.Normalize(to);
+                Vector2 range = to * 475;
+                trueCoords = current + range;
+            }
             else
-        trueCoords = Vector2:new(spell.X, spell.Y)
-            end
-        
-    addParticle(owner, "Ezreal_arcaneshift_cas.troy", owner.X, owner.Y);
-            teleportTo(owner, trueCoords.x, trueCoords.y)
-            addParticleTarget(owner, "Ezreal_arcaneshift_flash.troy", owner);
-
-            local target = nil
-            local units = getUnitsInRange(owner, 700, true)
-        
+            {
+                trueCoords = new Vector2(spell.X, spell.Y);
+            }
+            ApiFunctionManager.AddParticle(owner, "Ezreal_arcaneshift_cas.troy", owner.X, owner.Y);
+            spell.Teleport(owner, trueCoords.X, trueCoords.Y);
+            ApiFunctionManager.AddParticleTarget(owner, "Ezreal_arcaneshift_flash.troy", owner );
+            Unit target = null;
+            List<Unit> units = ApiFunctionManager.GetUnitsInRange(owner, 700, true);
```

Comment:
`owner.GetUnitsInRange(700)`. Remove the isAlive param, since that is rarely used, and instead opt to include that logic only in the places that actually use it.

(note: this might also be refactored to something like `game.GetUnitsInRange(owner, 700)`, which would probably be better since it assumes that only a single class is responsible for keeping track of entities throughout the game)